src/measure_theory/set_integral.lean

```diff
@@ -36,9 +36,14 @@ as `s` tends to `l.lift' powerset`, i.e. for any `ε>0` there exists `t ∈ l` s
 `∥∫ x in s, f x ∂μ - μ s • c∥ ≤ ε * μ s` whenever `s ⊆ t`. We also formulate a version of this
 theorem for a locally finite measure `μ` and a function `f` continuous at a point `a`.
 
-## Notation
+## Notations
```

Comment:
```suggestion
## Notation
```
just a weird English thing -- I wouldn't say "Notations" is wrong, but "Notation" is more right.